src/components/BeerDetails.js

```diff
@@ -0,0 +1,51 @@
+import { useParams } from "react-router-dom";
+import { NavLink } from "react-router-dom";
+
+function BeerDetails(props){
+
+    const {beerId} = useParams();
+
+  
+     const beersArr = props.beersList;
+     
+     
+     const details = beersArr.find((beer) => {
+        
+        return beer._id == beerId;
+
+     })
```

Comment:
one small thing: for the details they give you this endpoint: 

`GET | /:id -> Get the single/specific beer`
So they want you to make an axios request here as well. But no big deal, and still good job!!